macOS/DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -349,8 +352,9 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
         vpnSettings.alignTo(subscriptionEnvironment: subscriptionAuthV1toV2Bridge.currentEnvironment)
 
         // Update DBP environment and match the Subscription environment
-        let dbpSettings = DataBrokerProtectionSettings(defaults: .dbp)
+        let dbpSettings = DataBrokerProtectionSettings(defaults: .netP)
```

Comment:
Why is DBP to use netP's user defaults?

Reply:
I think GH is misbehaving O_o I don't have that! checking

Reply:
I think the GH disruptions are having an effect here, this is fixed and pushed from a while

Reply:
Now looks ok, closing